templates/verify.txt

```diff
@@ -1,6 +1,6 @@
-{{t "Almost there"}}
+{{t "Welcome!"}}
```

Comment:
@ryanfeeley i sent an email earlier about spam testing, but for now, based on tests from my local build, the ! is a push. SHIPIT

https://youtu.be/--Vaz9jW054